application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarEventSheet.xml

```diff
@@ -184,6 +186,20 @@ $doc.display($field)
     : $doc.display('allDay')
     ; &lt;label for="${editPrefix}_description"&gt;$escapetool.xml($doc.displayPrettyName('description', false, false))&lt;/label&gt;
     : $doc.display('description')
+    ; &lt;label for="${editPrefix}_backgroundColor"&gt;$escapetool.xml($doc.displayPrettyName('backgroundColor', false, false))&lt;/label&gt;
+    #set ($colorPickerParams = {
+      'id': "${editPrefix}_backgroundColor",
+      'name': "${editPrefix}_backgroundColor",
+      'value': $escapetool.xml($!doc.getValue('backgroundColor'))
```

Comment:
Why don't you use `$eventInstance.getBackgroundColor()`? You can initialize the `eventInstance` object so it can be accessed here as well

Reply:
When using ```$eventInstance.getBackgroundColor()```, the final displayed color is returned, perhaps replaced by the calendar color.

This means that when editing, the value of the field can be set to the current calendar default, overriding the actual value of the event color when saved. As a result, the color of an event will remain the same, not changing when the calendar default changes, even if it wasn't explicitly assigned a color by the user.